actions/custom.js

```diff
@@ -1,23 +1,37 @@
-import TimeZoneService from "../helpers/TimeZoneService";
 import {Platform} from 'react-native';
 import moment from "moment-timezone";
-
-const timezone = TimeZoneService.getTimeZone()
-
-export const customFormatLocaleDate = (date, format = null) => {
-  return moment(date ?? new Date()).tz(timezone).format(format ?? "YYYY-MM-DD HH:mm")
+import {api, dispatchFetchRequest} from "../constants/Backend";
+
+
+let setTimezone = null
```

Comment:
rename to selectedTimeZone.

Reply:
Renamed and set default value 'Asia/Taipei' to avoid errors on new devices.